src/win/winapi.c

```diff
@@ -63,6 +64,10 @@ void uv_winapi_init(void) {
     uv_fatal_error(GetLastError(), "GetModuleHandleA");
   }
 
+  pRtlGetVersion = (sRtlGetVersion) GetProcAddress(
```

Comment:
Registry key is documented here: https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development, RtlGetVersion is documented here: https://msdn.microsoft.com/en-us/library/windows/hardware/ff561910(v=vs.85).aspx